recipes/typeguard/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "typeguard" %}
+{% set version = "2.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b8ddc6e2e60bd64b7003f9a685a09ba387b74adf2f6bea7534a76d61892f573e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.4
+    - pip
+    - setuptools_scm
+  run:
+    - python >=3.4
+
+test:
+  imports:
+    - typeguard
+
+about:
+  home: https://github.com/agronholm/typeguard
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Runtime type checker for Python'
+  description: |
+    This library provides runtime type checking for functions defined with argument type annotations.
+  doc_url: https://github.com/agronholm/typeguard
+  dev_url: https://github.com/agronholm/typeguard
+
+extra:
+  recipe-maintainers:
+    - itamarst
+    - agronholm
```

Comment:
Great! @aaronang needs to confirm they are ok maintaining.

Reply:
I asked, see here: https://github.com/agronholm/typeguard/issues/56#issuecomment-456802335

Reply:
I think you mentioned the wrong person 😛

CC @agronholm

Reply:
Ack sorry!